legend-engine-pure-code-compiled-core/src/main/resources/core/external/format/openapi/tests.pure

```diff
@@ -0,0 +1,67 @@
e":"object","properties":{"package":{"$ref":"#/components/schemas/Package"},"referenceUsages":{"type":"array","items":[{"$ref":"#/components/schemas/ReferenceUsage"}]}}},"DoubleRangeProperty":{"type":"object","properties":{"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"Stereotype":{"type":"object","properties":{}},"ScriptLanguage":{"type":"object","properties":{"builtin":{"type":"string","enum":["painless","expression","mustache","java"]},"custom":{"type":"string"}}},"WildcardProperty":{"type":"object","properties":{"null_value":{"type":"string"},"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"DateNanosProperty":{"type":"object","properties":{"index":{"type":"boolean"},"ignore_malformed":{"type":"boolean"},"null_value":{"type":"string"},"boost":{"type":"float"},"format":{"type":"string"},"precision_step":{"type":"integer"},"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"TextProperty":{"type":"object","properties":{"index":{"type":"boolean"},"index_phrases":{"type":"boolean"},"search_analyzer":{"type":"string"},"analyzer":{"type":"string"},"index_options":{"type":"string","enum":["docs","freqs","positions","offsets"]},"norms":{"type":"boolean"},"term_vector":{"type":"string","enum":["no","yes","with_offsets","with_positions","with_positions_offsets","with_positions_offsets_payloads","with_positions_payloads"]},"type":{"type":"string"},"eager_global_ordinals":{"type":"boolean"},"fielddata_frequency_filter":{"$ref":"#/components/schemas/FielddataFrequencyFilter"},"position_increment_gap":{"type":"integer"},"boost":{"type":"float"},"fielddata":{"type":"boolean"},"index_prefixes":{"$ref":"#/components/schemas/TextIndexPrefixes"},"search_quote_analyzer":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"IndexTDS":{"type":"object","properties":{"index":{"$ref":"#/components/schemas/Elasticsearch7StoreIndexMapping"}}},"TokenCountProperty":{"type":"object","properties":{"index":{"type":"boolean"},"enable_position_increments":{"type":"boolean"},"null_value":{"type":"float"},"analyzer":{"type":"string"},"boost":{"type":"float"},"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"AggregateMetricDoubleProperty":{"type":"object","properties":{"default_metric":{"type":"string"},"metrics":{"type":"string"},"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"BinaryProperty":{"type":"object","properties":{"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"Package":{"type":"object","properties":{"children":{"type":"array","items":[{"$ref":"#/components/schemas/PackageableElement"}]}}},"RankFeaturesProperty":{"type":"object","properties":{"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}},"LatLonGeoLocation":{"type":"object","properties":{"lon":{"type":"float"},"lat":{"type":"float"}}},"DateRangeProperty":{"type":"object","properties":{"format":{"type":"string"},"type":{"type":"string"},"_pure_protocol_type":{"type":"string"}}}}}}';
```

Comment:
Can we put this on a file on the pure repo, and then use readFile function to get it?

Then, we can use assertJsonEquals function.

This would help with readability.

Reply:
@rafaelbey Done. Can you please review and approve this?